src/app/pages/BooksPage/slice/index.ts

```diff
@@ -0,0 +1,79 @@
+import { createSlice } from '@reduxjs/toolkit';
+import { PayloadAction } from '@reduxjs/toolkit';
+import { useInjectReducer, useInjectSaga } from 'utils/redux-injectors';
+import { BookState } from './types';
+import { rootBookSaga } from './saga';
+import { injectReducer, injectSaga } from 'redux-injectors';
+
+export const initialState: BookState = {
```

Comment:
This is not the correct way to store a list of books. This state assumes that there is a single book to be stored.